### ISSUE TITLE
Low ES6 support

### ISSUE THREAD

**Issue by jlam55555** [jaywunder/music-visualizer#2]
This looked interesting, so I started looking over your code. In your code, there are many ES6-only constructs, such as `const` and `class`. These have little-to-no support in browsers like Opera or IE.

As a result, got this error running your project:

> Uncaught SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode

Perhaps look into [Babel](https://babeljs.io/) or [TypeScript](https://www.typescriptlang.org/)? Or at least add `"use strict";` to allow it in more browsers?

**Comment by junewunder** [comment]
Yeah that's planned in the future, not now, I'm probably going to learn webpack with this project. It's the summer right now so I'm feeling productive occasionally but not too often.  Although summer is almost done so you'll probably see some more things happening here soon!  

For now, if you want to test it then use chrome 😛  but it's not too interesting for the moment. 

Thanks for checking out my project!
